Code/Editor/EditorEngineProcessFramework/LongOps/Implementation/LongOpControllerManager.cpp

```diff
@@ -64,18 +64,18 @@ void ezLongOpControllerManager::StartOperation(ezUuid opGuid)
   BroadcastProgress(*pOpInfo);
 }
 
-void ezLongOpControllerManager::CancelOperation(ezUuid opGuid)
+void ezLongOpControllerManager::CancelOperation(ezUuid guid)
```

Comment:
Interesting, did it change that automatically?

Reply:
Yes, this is a very recent issue I fixed. In this case clang-tidy identified 'op' as a type prefix and removed it. The latest version of clang-tidy will no longer do that. Because it was a very signifanct amount of work to make everything compile again after doing a full clang-tidy run, I didn't want to revert and re-run. I manually reverted this change (and others like it)